yt_dlp/extractor/filmarchiv.py

```diff
@@ -0,0 +1,45 @@
+from .common import InfoExtractor
+
+
+class FilmArchivIE(InfoExtractor):
+    IE_NAME = 'FILMARCHIV ON'
+    _VALID_URL = r'https?://(?:www\.)?filmarchiv\.at/(?:de|en)/filmarchiv-on/video/(?P<id>[0-9a-zA-Z_]+)'
+    _TESTS = [{
+        'url': 'https://www.filmarchiv.at/de/filmarchiv-on/video/f_0305p7xKrXUPBwoNE9x6mh',
+        'md5': 'TODO: md5 sum of the first 10241 bytes of the video file (use --test)',
+        'info_dict': {
+            'id': 'f_0305p7xKrXUPBwoNE9x6mh',
+            'ext': 'mkv',
+            'title': 'Der Wurstelprater zur Kaiserzeit',
+            'description': 'md5:9843f92df5cc9a4975cee7aabcf6e3b2',
+            'thumbnail': 'https://img.filmarchiv.at/unsafe/1024x1024/videostatic/f_0305/p7xKrXUPBwoNE9x6mh_v1/poster.jpg',
+        },
+    }]
+
+    def _real_extract(self, url):
+        media_id = self._match_id(url)
+        webpage = self._download_webpage(url, media_id)
+
+        title = self._html_search_regex(
+            r'<title-div [^>]+>\s*(.+?)\s*</title-div>',
+            webpage, 'title')
+
+        description = self._html_search_regex(
+            r'<div class="(?:.+?)?border-base-content[^"]*">\s*<div class="(?:.+?)?prose[^"]*">\s*<p>\s*(.+?)\s*</p>',
+            webpage, 'description')
```

Comment:
```suggestion
        description = traverse_obj(webpage, (
            {find_elements(
                tag='div',
                attr='class', value=r'[^\'"]*(?<=[\'"\s])border-base-content(?=[\'"\s])[^\'"]*',
                html=False, regex=True)}, ...,
            {find_elements(
                tag='div',
                attr='class', value=r'[^\'"]*(?<=[\'"\s])prose(?=[\'"\s])[^\'"]*',
                html=False, regex=True)}, ...,
            {clean_html}, any,
        ))
```
remember to add imports